pyliquibase/__init__.py

```diff
@@ -12,21 +17,31 @@
 handler.setFormatter(formatter)
 log.addHandler(handler)
 
-
 #####
 
+LAST_STABLE_KNOWN_LB_VERSION: str = "4.6.2"
+COMPATIBLE_LIQUIBASE_VERSIONS: List = ["4.6.1", "4.6.2"]
```

Comment:
i think we dont need this? it will be dificult to keep it up to date. lets make user to provide correct version.